### ISSUE TITLE
Shebang format + config file convention for Fedora packaging

### ISSUE THREAD

**Issue by AquaL1te** [dylanaraps/neofetch#762]
## Description

I'm packaging Neofetch for Fedora and the fedora-review package encountered a few things:
```
Checking: neofetch-3.2.0-1.fc27.noarch.rpm
          neofetch-3.2.0-1.fc27.src.rpm
neofetch.noarch: E: wrong-script-interpreter /usr/bin/neofetch /usr/bin/env bash
neofetch.noarch: W: non-conffile-in-etc /etc/neofetch/config
neofetch.noarch: E: wrong-script-interpreter /etc/neofetch/config /usr/bin/env bash
neofetch.noarch: E: non-executable-script /etc/neofetch/config 644 /usr/bin/env bash
2 packages and 0 specfiles checked; 3 errors, 1 warnings.
```

1) I changed the _/usr/bin/env bash_ in Neofetch to _/usr/bin/bash_ with a sed command in the [spec file](https://github.com/AquaL1te/neofetch/commit/d2937d93a94df82fdb5444fdffd587abbf14078a) of the RPM.
2) I also removed the _/usr/bin/env bash_ from the config file since it's [sourced](https://github.com/dylanaraps/neofetch/blob/master/neofetch#L3604) anyway.
3) Would it be possible to add the .conf extension to the config file? It won't break functionality, it's just a uniformity thing.

So in a nutshell; is the shebang in the config necessary? And could you please add the .conf extension to the config file? The most important bit is the question about the shebang in the config file because I don't want to break the functionality. The other errors generated by the fedora-review tool can either be ignored or fixed in the spec file.

**Comment by dylanaraps** [comment]
See #763

**Comment by dylanaraps** [comment]
The changes have been made. :)
